e2e/tests/core/02-client/client_test.go

```diff
@@ -71,6 +73,64 @@ func (s *ClientTestSuite) QueryAllowedClients(ctx context.Context, chain ibc.Cha
 	return res.Params.AllowedClients
 }
 
+// TestScheduleIBCUpgrade_Succeeds tests that a governance proposal to schedule an IBC software upgrade is successful.
+func (s *ClientTestSuite) TestScheduleIBCUpgrade_Succeeds() {
+	t := s.T()
+	ctx := context.TODO()
+
+	_, _ = s.SetupChainsRelayerAndChannel(ctx)
+	chainA, chainB := s.GetChains()
+	chainAWallet := s.CreateUserOnChainA(ctx, testvalues.StartingTokenAmount)
+
+	var (
+		originalChainId string
+		newChainId      string
```

Comment:
is this upgrading the chain to have an empty chainID? We should set some value?

Reply:
looks like it

Reply:
I think this should be a constant defined here

Reply:
I asked @charleenfei to update this to pull the current chainID and increment its revision number (as this is the recommended approach to changing chainIDs)